README.md

```diff
@@ -37,32 +37,32 @@ For simplicity's sake, the tool is just a single script you paste into a UI Auto
 
 At the top of the script, you'll see a JavaScript dictionary of configuration settings:
 
-    config: {
-      numberOfEvents: 1000,
-      delayBetweenEvents: 0.05,    // In seconds
-
-      // Events are triggered based on the relative weights here.
-      // The event with this highest number gets triggered the most.
-      eventWeights: {
-        tap: 30,
-        drag: 1,
-        flick: 1,
-        orientation: 1,
-        clickVolumeUp: 1,
-        clickVolumeDown: 1,
-        lock: 1,
-        pinchClose: 10,
-        pinchOpen: 10,
-        shake: 1
-      },
-
-      // Probability that touch events will have these different properties
-      touchProbability: {
-        multipleTaps: 0.05,
-        multipleTouches: 0.05,
-        longPress: 0.05
-      }
-    },
+	config: {
```

Comment:
Out of curiosity, why was this indented further? Since it doesn't look like the contents significantly changed, can we get rid of the change in whitespace so there's less unnecessary change in this PR?